README.md

```diff
@@ -1,13 +1,46 @@
-# Sample Hardhat Project
+# Weekend Project 2: Ballot Scripts
 
-This project demonstrates a basic Hardhat use case. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
+## Development
 
-Try running some of the following tasks:
+```bash
+yarn install
+yarn hardhat compile
+```
+
+Then create a file named `.env` with the following content and fill values:
 
-```shell
-npx hardhat help
-npx hardhat test
-REPORT_GAS=true npx hardhat test
-npx hardhat node
-npx hardhat run scripts/deploy.ts
+```env
+ALCHEMY_API_KEY=
+GOERLI_PRIVATE_KEY=
+BALLOT_ADDRESS=0x79eEcC3b5e50598129EaF64f7f86f65458E071Cd
 ```
+
+## Scripts
+
+### Create Vote
+
+```bash
+yarn hardhat createVote --account 0xD9aF6C670B49C4b1239B86bb472E877f5BdF13Bf
+```
+
+Example TX: https://goerli.etherscan.io/tx/0x6baeef448ff447d6cfaf0cf1f58deb73138d11835021ff8beeb0297193386a0c
+
+### Give Right to Vote
+
+```bash
+ yarn hardhat giveRightToVote --account 0xD9aF6C670B49C4b1239B86bb472E877f5BdF13Bf
+```
+
+_Note: Only `chairman`(deployer) can call this script_
+
+Example TX: TODO
+
+## Helper Fucntions
+
+### Get Ballot Contract
+
+This function creates an instance of `Ballot` contract using `BALLOT_ADDRESS` variable in the `.env` file as address and `accounts[0]` as signer.
+
+### Get Etherscan Tx Link
+
+This function returns `Etherscan` link to Tx.
```

Comment:
Shoud we add how to deploy it Goerli Test Net?
yarn hardhat run scripts/deploy.ts --network goerli

Reply:
Since I don't want to mess up with deploy.ts, I created another deployBallot.ts that will deploy Ballot to Goerli testnet